backend/src/services/smtp/smtp.ts

```diff
@@ -95,13 +96,19 @@ export async function notifyReviewResponseForRelease(review: ReviewDoc, release:
     log.info('Not sending email due to SMTP disabled')
     return
   }
-  const reviewResponse = review.responses[0]
+
+  const reviewResponse = await findResponseById(review.responseIds[0])
```

Comment:
I don't think this is accurate anymore.  With `comments` this might not represent the latest decision.  I think this _might_ need to be the last item in the array.